stress/frontend/config/stressTestInfra.conf

```diff
@@ -0,0 +1,3 @@
+sweetSpot = 14 //maximum throughput the backend can handle
+minRate = 1    //the minimum throughput when there is too litte or too much work
+kanaloa: true  //run test with kanaloa?
```

Comment:
Should use `=` for consistency, and `#` for comments (`//` works too, but we use `#` elsewhere)

Also configs usually go in `/src/main/resources/something.conf` and can be loaded with `ConfigFactory.parseResources("something.conf")`